ocs_ci/utility/environment_check.py

```diff
@@ -82,8 +85,15 @@ def assign_get_values(env_status_dict, key, kind=None):
             copy.deepcopy(ENV_STATUS_DICT)
         key (str): Name of the resource
         kind (OCP obj): OCP object for a resource
+        exclude_namespaces (list): List of namespaces to ignore
```

Comment:
exclude_namespaces is tuple right ?

Reply:
I guess that the default value is tuple because it's not best practice (and it's dangerous) to have mutable objects as default value. But this is just default value and you can pass list here and with tuple it will also work.

Reply:
I meant doc string says its a list but actually passing tuple, so its a typo.

Reply:
Yeah actually what Petr said is the reason for passing tuple and documenting as a list.
Passing mutable as default raise a warning (flake 8).

I think that's ok like it is now

Reply:
We could default `exclude_namespaces` to `None` and set to `['openshift-marketplace']` if it is `None`. This way we don't deal with tuples at all and the default value doesn't "contradict" the type.

```python
def assign_get_values(
    env_status_dict, key, kind=None,
    exclude_namespaces=None
):
    exclude_namespaces = exclude_namespaces or ['openshift-marketplace']

Reply:
Either document in docstring as List or Tuple or we can use what Coady suggeted to move ahead with this PR and resolve comments.

Reply:
fixed